R/RLearner_regr_gbm.R

```diff
@@ -28,13 +30,25 @@ makeRLearner.regr.gbm = function() {
 #' @export
 trainLearner.regr.gbm = function(.learner, .task, .subset, .weights = NULL, ...) {
   f = getTaskFormula(.task)
-  if (is.null(.weights)) {
-    f = getTaskFormula(.task)
-    gbm::gbm(f, data = getTaskData(.task, .subset), ...)
+
+  params = list(...)
+  if("alpha" %in% names(params)) {
```

Comment:
Space after if and I would check whether `params$alpha` is `NULL` for consistency with how it's set below.